src/main/java/pl/patrykks/services/impl/DefaultFacebookService.java

```diff
@@ -50,15 +50,15 @@ public Facebook findById(String id) throws NotFoundException {
                 collect(Collectors.toSet());
     }
 
-    public Set<Facebook> findAll() {
+    public SortedSet<FacebookProfile> findAll() {
         logger.debug("Finding all stored facebook profiles");
-        Comparator<Facebook> personComparator = Comparator.comparing(Facebook::getLastname)
-                .thenComparing(Comparator.comparing(Facebook::getFirstname));
+        Comparator<FacebookProfile> personComparator = Comparator.comparing(FacebookProfile::getLastname)
```

Comment:
I'm not sure if the sorting order is correct. I would like to ask if the set of Facebook profiles returned by service should be sorted by lastname and then by firstname, or first by firstname and then by lastname.